ecosystem/sep-0006.md

```diff
@@ -144,6 +154,8 @@ Name | Type | Description
 `amount` | string | (optional) The amount of the asset the user would like to deposit with the anchor. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
 `country_code` | string | (optional) The [ISO 3166-1 alpha-3](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) code of the user's current address. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
 `claimable_balance_supported` | string | (optional) `true` if the client supports receiving deposit transactions as a claimable balance, `false` otherwise.
+`destination_asset` | string | (optional) The on-chain asset the Anchor will deliver to the user's Stellar account. The value must match one of the `asset` values included in a [`SEP-38 GET /prices?sell_asset=iso4217:<asset_code>`](sep-0038.md#get-prices) response using [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format). If neither this field nor `quote_id` are set, it's assumed the Anchor will be providing a bridge between equivalent assets, for instance receiving fiat USD and depositing USDC to the user's Stellar account.
+`quote_id` | string | (optional) The `id` returned from a `SEP-38 POST /quote` response. If this attribute is specified, none of the attributes `asset_code`, `amount` or `destination_asset` should be included in the request. The Stellar transaction used to send the asset _must_ have a `started_at` timestamp that is earlier than the quote's `expires_at` attribute.
```

Comment:
> The Stellar transaction used to send the asset _must_ have a `started_at` timestamp that is earlier than the quote's `expires_at` attribute.

❗  This is not true, since the `expires_at` attribute from the quote describes the time by which the user must deliver funds to the anchor, not the opposite. If the user has delivered funds to the anchor before the quote's expiration is should be acceptable for the anchor to deposit funds to the user's stellar account after the expiration.

Reply:
You're right, that part of the description didn't make much sense for the deposit. Here's the fix: https://github.com/stellar/stellar-protocol/pull/973/commits/3c832f6ce361e397d3632949f2a1bfc756915649